### PR TITLE
Add link to napari/docs in Readme file. 

### PR DESCRIPTION
**Pull request by Nadalyn-CZI**
# Fixes/Closes

Closes #

# Description
Adds a link to napari/docs in Readme. 

cc @dgmccart

# References

## Type of change

- [ ] Bug-fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How has this been tested?

- [ ] example: the test suite for my feature covers cases x, y, and z
- [ ] example: all tests pass with my change
- [ ] example: I check if my changes works with both PySide and PyQt backends
      as there are small differences between the two Qt bindings.  

## Final checklist:
- [x] My PR is the minimum possible work for the desired functionality
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I included new strings, I have used `trans.` to make them localizable.
      For more information see our [translations guide](https://napari.org/developers/translations.html).

### CODE REVIEW

**Comment by GenevieveBuckley** [comment]
I think the first link people should see is the one to the docs at https://napari.org. I don't think it makes sense to remove that, since it should be one of the first things people look at.

Second, there is also a ["tutorials" section of the README](https://github.com/napari/napari/#tutorials) that will also need updating. Perhaps that, or the ["contributing" section](https://github.com/napari/napari/#contributing) are better places to put the information and link to https://github.com/napari/docs

**Comment by Nadalyn-CZI** [comment]
Genevieve, I added that information back in.  I agree with you.  I moved the information about contributing to the contributing section. I don't know what needs to be updated in the tutorial section.  It basically refers people to the tutorials on napari.org.

**Review by psobolewskiPhD — APPROVED** [review]
Thanks @Nadalyn-CZI

**Comment by Nadalyn-CZI** [comment]
@psobolewskiPhD I made one final change. (There's almost always a way to make a doc a little bit better!)  Please approve this again, if you think it's worthy. 

Thanks, 
Nadalyn

File: README.md
Comment (DragaDoncila) [inline README.md]:
```suggestion
Contributions are encouraged! Please read our [contributing guide](https://napari.org/developers/contributing.html) to get started. Given that we're in an early stage, you may want to reach out on our [Github Issues](https://github.com/napari/napari/issues) before jumping in. 

If you want to contribute or edit to our documentation, please go to [napari/docs](https://github.com/napari/docs). 
```

**Review by DragaDoncila — APPROVED** [review]
This looks fine to me, definitely need to be mentioning the docs repo in the README.

**Comment by DragaDoncila** [comment]
Docs build was failing so I've reran checks (might take a while). The doc build error looked unrelated (something about `tomli_w` and `npe2` docs), but we should make sure before merge.

**Pull request merged**
